src/Enemy/EnemyStateWander.cpp

```diff
@@ -0,0 +1,110 @@
+#include "Enemy/EnemyStateWander.h"
+#include <al/Library/LiveActor/ActorActionFunction.h>
+#include <al/Library/LiveActor/ActorCollisionFunction.h>
+#include <al/Library/LiveActor/ActorMovementFunction.h>
+#include <al/Library/LiveActor/ActorPoseKeeper.h>
+#include <al/Library/Math/MathRandomUtil.h>
+#include <al/Library/Nerve/NerveSetupUtil.h>
+#include <al/Library/Nerve/NerveUtil.h>
+
+namespace {
+NERVE_IMPL(EnemyStateWander, Wait);
+NERVE_IMPL(EnemyStateWander, Walk);
+NERVE_IMPL(EnemyStateWander, Fall);
+
+struct {
+    NERVE_MAKE(EnemyStateWander, Wait);
+    NERVE_MAKE(EnemyStateWander, Walk);
+    NERVE_MAKE(EnemyStateWander, Fall);
+} NrvEnemyStateWander;
+
+}  // namespace
+
+EnemyStateWander::EnemyStateWander(al::LiveActor* actor, const char* stateName) : al::ActorStateBase("さんぽ状態", actor) {
+    mRandNum = 0;
+    mWalkSpeed = -1.0f;
+    mStateName = stateName;
+    mIsHalfProbability = false;
```

Comment:
define in header

Reply:
Styling, and having the default values written in the header next to the declaration instead of hiding it somewhere in the actual source.